notebooks/samples/305 - Flowers ImageFeaturizer.ipynb

```diff
@@ -0,0 +1,461 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run the experiment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+   },
+   "outputs": [],
+   "source": [
+    "# Increase or remove the sampling in order to get better results\n",
+    "train, test = imagesWithLabels.sample(False, 0.03).randomSplit([.8,.2])\n",
+    "train, test = train.repartition(1), test.repartition(1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train.cache()\n",
+    "test.cache()\n",
+    "train.count(), test.count()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = TrainClassifier().setModel(LogisticRegression()).setLabelCol(\"labels\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "trainFeatsBasic, testFeatsBasic = featurize(basicFeaturizer,train,test,'basic')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "basicPredictions = predict(model, trainFeatsBasic, testFeatsBasic, 'basic')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "trainFeatsCNTK, testFeatsCNTK = featurize(cntkFeaturizer,train,test,'cntk')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cntkPredictions = predict(model, trainFeatsCNTK, testFeatsCNTK, 'cntk')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "mml-deploy": "local"
+   },
+   "outputs": [],
+   "source": [
+    "basicPredictions = basicPredictions.toPandas()\n",
+    "cntkPredictions = cntkPredictions.toPandas()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "mml-deploy": "hdinsight"
+   },
+   "outputs": [],
+   "source": [
+    "basicPredictions.registerTempTable(\"basicPredictions\")\n",
+    "cntkPredictions.registerTempTable(\"cntkPredictions\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "mml-deploy": "hdinsight"
+   },
+   "outputs": [],
+   "source": [
+    "%%sql -q -o basicPredictions\n",
+    "select * from basicPredictions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "mml-deploy": "hdinsight"
+   },
+   "outputs": [],
+   "source": [
+    "%%sql -q -o cntkPredictions\n",
+    "select * from cntkPredictions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot confusion matrix."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "aml-deploy": "hdinsight",
```

Comment:
This cell should work on both HDI and local - could you please remove the metadata tag and check?